### PR TITLE
Add modules to project

### PR DESCRIPTION
**Pull request by Luchy-B**
-Add modules 
- Use arrow functions

### CODE REVIEW

File: modules/date.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (OLIPLICHE) [inline modules/date.js]: - [ ]  I would advise you to solve the Linter errors Manually and not disable it.

- Solve Linters errors or bugs, manually helps you to become an expert in debugging code. you can use [stackoverflow](https://stackoverflow.com/) website to search for how to solve bugs you are facing.

File: modules/navigation.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (OLIPLICHE) [inline modules/navigation.js]: - [ ]  I would advise you to solve the Linter errors Manually and not disable it.

- Solve Linters errors or bugs, manually helps you to become an expert in debugging code. you can use [stackoverflow](https://stackoverflow.com/) website to search for how to solve bugs you are facing.

File: modules/book.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (OLIPLICHE) [inline modules/book.js]: - [ ]  I would advise you to solve the Linter errors Manually and not disable it.

- Solve Linters errors or bugs, manually helps you to become an expert in debugging code. you can use [stackoverflow](https://stackoverflow.com/) website to search for how to solve bugs you are facing.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Luchy-B, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️

- [ ] I would advise you to use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6) to display the current date below the navigation bar. This is part of the project requirements. Also, Project requirement is not all about Microvese, but in near future at the workplace, you will be working based on the client's requirement, and here is the best time to learn how to follow the requirements

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (FrederickMih) [inline LICENSE.md:1]: - [Optional] **MIT** license link in your README.md file is not working. Please do the fix in order to render your README file professional 👍

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @Luchy-B , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights </img>


- PR is professional ✔️ 
- Date display as expected using Luxon library 👍 
- Linters checks passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: LICENSE.md (hunk 4 above)
Comment (Luchy-B) [reply LICENSE.md:1]: MIT link fixed.

Thank you for the observation/Review

**Pull request merged**